src/ostorlab/cli/rootcli.py

```diff
@@ -49,3 +52,6 @@ def rootcli(
         loggers = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
         for l in loggers:
             l.setLevel(logging.DEBUG)
+    if daemon is True:
+        start_daemon = dm.DaemonContext()
```

Comment:
What does it call?

Reply:
![image](https://github.com/Ostorlab/ostorlab/assets/4823598/f0ae5f40-21f7-473f-bb97-8b56041e102c)